modules/tinymce/src/core/test/ts/browser/EditorCleanupTest.ts

```diff
@@ -0,0 +1,47 @@
+import { Assertions, Chain, Log, NamedChain, Pipeline } from '@ephox/agar';
+import { Editor as McEditor } from '@ephox/mcagar';
+import { UnitTest } from '@ephox/bedrock-client';
+import { Attr, Element, Truncate } from '@ephox/sugar';
+import { HTMLDivElement } from '@ephox/dom-globals';
+
+import Editor from 'tinymce/core/api/Editor';
+import { RawEditorSettings } from 'tinymce/core/api/SettingsTypes';
+import Theme from 'tinymce/themes/silver/Theme';
+import VisualBlocksPlugin from 'tinymce/plugins/visualblocks/Plugin';
+
+UnitTest.asynctest('browser.tinymce.core.EditorCleanupTest', (success, failure) => {
```

Comment:
No tests here really check that the attributes are preserved, as you're always almost always just using an empty div with no attributes. The one exception is the `placeholder` attribute, but it'd be good to test other things like style, data attributes, etc...